addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-service.js

```diff
@@ -487,6 +491,141 @@ class EnvironmentScService extends Service {
     return result;
   }
 
+  /**
+   * Updates the study ids for the environment sc entity. IMPORTANT: do NOT use this from an api handler, this method
+   * does NOT validate permissions to the study ids provided. If you need to update the study for an environment with
+   * permissions enforced, then use the studyOperationService.updatePermissions() method.
+   *
+   * @param requestContext The standard request context
+   * @param studyIds An array of study ids
+   */
+  async updateStudyIds(requestContext, id, studyIds = []) {
```

Comment:
This method was not called anywhere? Also, why do we need a separate method to update studyIds specifically instead of using the generic update method that already exist?

Reply:
Removed.
I used to call it from the study registration service, I then commented out the code in the study registration service and thought I would leave it for future use.  The main reason for not having it done in the update method is that it meant that we needed to updated the 'update schema' and allow the 'studyIds' to be updated. This meant that we needed to add validation logic that checks the permission for each study id against the creator of the environment not the request context, basically it was going to complicate the update method.  Anyway, I think removing it for now is a good thing.  Done.